Source/WebKit/WebProcess/GPU/GPUProcessConnection.h

```diff
@@ -66,7 +66,7 @@ class RemoteVideoFrameObjectHeapProxy;
 
 class GPUProcessConnection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<GPUProcessConnection>, public IPC::Connection::Client {
 public:
-    static RefPtr<GPUProcessConnection> create(IPC::Connection& parentConnection);
+    static Ref<GPUProcessConnection> create(Ref<IPC::Connection>);
```

Comment:
We should not pass smart pointer by value.

Reply:
The idea is to communicate the pass of the ownership of the reference, similar to when passing unique_ptr.

Reply:
This is what `&&` is for. Please follow our patterns.